### PR TITLE
changes necessary to enable the correct extras repo on rhel 7/8

### PR DESCRIPTION
**Pull request by bodanc**

### CODE REVIEW

**Comment by bodanc** [comment]
https://reynencourt.atlassian.net/browse/PD-156

testing done; all good;

File: roles/container-engine/containerd/defaults/main.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ containerd_repo_key_info:
 containerd_repo_info:
   repos:
 
-extras_rh_repo_base_url: "http://mirror.centos.org/centos/$releasever/extras/$basearch/"
-extras_rh_repo_gpgkey: "http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7"
+#extras_repo_base_url: "http://mirror.centos.org/centos/$releasever/extras/$basearch/"
+#extras_repo_gpgkey: "http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7"
```
Comment (r3dsm0k3) [inline roles/container-engine/containerd/defaults/main.yml]: Can you remove the commented out code if it is not needed?

File: roles/container-engine/containerd/tasks/containerd_repo.yml [hunk 2]
```diff
@@ -45,26 +45,26 @@
...
-    baseurl: "http://mirror.centos.org/centos/{{ansible_distribution_major_version}}/extras/{{ansible_architecture}}/"
-    file: "extras"
-    gpgcheck: "{{ 'yes' if extras_rh_repo_gpgkey else 'no' }}"
-    gpgkey: "{{ extras_rh_repo_gpgkey }}"
-    keepcache: "{{ containerd_rpm_keepcache | default('1') }}"
-    proxy: " {{ http_proxy | default('_none_') }}"
-  when:
-    - ansible_distribution in ["CentOS","RedHat"]
-    - yum_result.results | length == 0
+#- name: check if container-selinux is available
```
Comment (r3dsm0k3) [inline roles/container-engine/containerd/tasks/containerd_repo.yml]: Can you remove the commented out code?

**Review by r3dsm0k3 — CHANGES_REQUESTED** [review]
@bodanc There's some code I see as commented out. We should _remove_ it if it is not needed to achieve cleanliness. Comments does not add any value in this case. Git always preserves the history, so even in the case you want to refer later you can always go back and check it.

**Pull request merged**